extensions/wikia/FacebookClient/tests/FacebookClientTest.php

```diff
@@ -7,6 +7,15 @@ public function setUp() {
 	}
 
 	public function testGetReturnToUrl() {
+
+		// TODO: remove this once FB is turned on globally and unconditionally
```

Comment:
@garthwebb Should we wait for this until [UC-209](https://wikia-inc.atlassian.net/browse/UC-209)? Otherwise, test environments keep failing left and right because of a WF variable not set.